snmp/tests/common.py

```diff
@@ -189,8 +189,8 @@
     }
 ]
 
-agent_autodiscovery_only = pytest.mark.skipif(AUTODISCOVERY_TYPE != 'agent', reason='Agent discovery only')
-python_autodiscovery_only = pytest.mark.skipif(AUTODISCOVERY_TYPE != 'python', reason='Python discovery only')
+snmp_listener_only = pytest.mark.skipif(SNMP_LISTENER_ENV != 'true', reason='Agent snmp lister tests only')
+snmp_integration_only = pytest.mark.skipif(SNMP_LISTENER_ENV != 'false', reason='Normal tests')
```

Comment:
should this specify which integration? From its standing point python is the only integration, from the dev standing point could raise the question

Reply:
When it's `snmp_integration_only`, it's running some tests for both integrations, we are running for:
- python integration the unit, integration, e2e tests
- core the e2e tests